src/tables/components/table-caption.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function CaptionComponent({ caption, className }) {
+  return <caption className={className}>{caption}</caption>;
```

Comment:
We don't have prettier on this repo (yet!!! #510), so could you remove the semicolons manually please?